plugin/auto/setup_test.go

```diff
@@ -75,6 +75,13 @@ func TestAutoParse(t *testing.T) {
 			}`,
 			true, "/tmp", "bliep", `(.*)`, 10 * time.Second, nil,
 		},
+		// no template specified.
+		{
+			`auto {
+				directory /tmp (.*)
+			}`,
+			true, "/tmp", "bliep", `(.*)`, 60 * time.Second, nil,
```

Comment:
why "bliep"?

Reply:
Well, actually it doesn't matter because we're expecting an error here (`shouldErr = true`). I just reused another test case. Anyway, I'll remove `bliep` to make it clear.

Reply:
Thanks! lgtm